content/en/docs/_index.adoc

```diff
@@ -1,62 +1,59 @@
----
-title: "Documentation"
-linkTitle: "Documentation"
-draft: false
-weight: 20
-no_list: true
-menu:
-  main:
-    weight: 20
----
-
-
+= Documentation
+:draft: false
+:linkTitle: Documentation
+:menu: {"main"=>{"weight"=>20}}
+:no_list: true
+:weight: 20
```

Comment:
This is Hugo front matter - I believe Hugo expects this to exist in YAML format somewhere in the source content. Can you try reverting this to see what happens?

Reply:
> This is Hugo front matter - I believe Hugo expects this to exist in YAML format somewhere in the source content. Can you try reverting this to see what happens?

I've restored the YAML frontmatter in all the files. The TOC and header all look good now.
However, I'm seeing some other issues with the content that must be fixed before this PR is ready to merge.